node/main.go

```diff
@@ -384,24 +384,25 @@ func main() {
 		return
 	}
 
+	if nodeConfig.Engine.DataWorkerBaseListenMultiaddr == "" {
+		nodeConfig.Engine.DataWorkerBaseListenMultiaddr = "/ip4/127.0.0.1/tcp/%d"
+	}
+	if nodeConfig.Engine.DataWorkerBaseListenPort == 0 {
+		nodeConfig.Engine.DataWorkerBaseListenPort = 40000
+	}
+	if nodeConfig.Engine.DataWorkerMemoryLimit == 0 {
+		nodeConfig.Engine.DataWorkerMemoryLimit = 1792 * 1024 * 1024 // 1.75GiB
```

Comment:
Do we want to keep the 1.75 GiB default per worker memory limit ? In my experiments I've never seen it use _that much_ memory. Should we lower this to 1GiB perhaps ? Or even lower maybe.

Reply:
it will increase once workers are responsible for their own management of state

Reply:
While you are making this change, it would be nice to keep a ENV variable that could set the number of workers, as it would require less change for anyone using GOMAXPROCS to manage workers.